robottelo/ui/settings.py

```diff
@@ -45,10 +44,10 @@ def update(self, tab_locator, param_name, value_type, param_value):
         self.click((strategy, value % param_name))
 
         if value_type == 'dropdown':
-            Select(
-                self.find_element(locators['settings.select_value'])
-            ).select_by_value(param_value)
+            self.wait_until_element(locators['settings.select_value'])
```

Comment:
That waiter is redundant.
Please, fix base class and change:
`element = self.find_element(locator)`
on
`element = self.wait_until_element(locator)`